src/avatar/Avatar.tsx

```diff
@@ -0,0 +1,24 @@
+import "./_avatar.scss";
+
+import React from "react";
+import classNames from "classnames";
+
+export interface AvatarProps {
+  src: string;
+  alt: string;
+  size?: number;
+  customClassName?: string;
+}
+
+function Avatar({alt, size, src, customClassName}: AvatarProps) {
+  return (
+    <img
+      src={src}
+      alt={alt}
+      style={{width: `${size}px`, height: `${size}px`}}
```

Comment:
What happens if `size` is undefined ? Does it work as expected ? I think we should check if the size is truthy before assigning the style object